src/main/java/com/hartwig/platinum/kubernetes/SampleArgument.java

```diff
@@ -15,28 +13,13 @@ public interface SampleArgument {
 
     Map<String, String> arguments();
 
-    static SampleArgument sampleJson(final TumorNormalPair pair, final String runName) {
-        return ImmutableSampleArgument.builder()
-                .id(pair.name().toLowerCase())
-                .putArguments("-sample_json", format("samples/%s-%s", pair.name().toLowerCase(), runName))
-                .putArguments("-set_id", pair.name())
-                .putArguments("-run_id", runName + pair.tumorIndex().map(ti -> "-" + ti).orElse(""))
```

Comment:
you'll notice I've removed the tumor index without a replacement. The tumor indexing is done so we can support the use case of running multiple tumor samples against one ref without a lot of duplicate config. This was done for CCIA, who will no longer use platinum. I think we can remove this functionality, always possible to add it back later.